public/mocks/v1_get_place_node_attributes.json

```diff
@@ -0,0 +1,767 @@
+{
+  "data": {
+    "summary": "In 2015, Querência exported 560,097 tons of soy occupying a total of 176,131ha of land. With 1% of the total production, it ranks 121st in Brazil in soy production, and 2nd in the state of Mato Grosso. The largest exporter of soy in Querência was Cargill, which accounted for 31% of the total exports, and the main destination was China.",
+    "area": 17786.2,
+    "biome_geoId": "BR1",
+    "biome_name": "AMAZONIA",
+    "column_name": "MUNICIPALITY",
+    "country_geoId": "BR",
+    "country_name": "BRAZIL",
+    "farming_GDP": 0.506,
+    "municip_geoId": "BR5107065",
+    "municip_name": "QUERENCIA",
+    "soy_farmland": 0.7340000000000001,
+    "state_geoId": "BR51",
+    "state_name": "MATO GROSSO",
+    "indicators": [
+      {
+        "name": "Key sustainability indicators",
+        "includedColumns": [
+          {
+            "name": "Soy deforestation (until 2014)",
+            "unit": "ha"
+          },
+          {
+            "name": "Number of environmental embargoes (2015)"
+          },
+          {
+            "name": "Area affected by fires (2013)",
+            "unit": "km2"
+          },
+          {
+            "name": "Cases of forced labor (2014)"
+          },
+          {
+            "name": "Cases of land conflicts (2014)"
+          },
+          {
+            "name": "Biodiversity (loss of amphibian habitat)"
+          },
+          {
+            "name": "Annual deforestation rate (2015)",
+            "unit": "ha"
+          },
+          {
+            "name": "Human development index (2013)"
+          },
+          {
+            "name": "Criticality of water scarcity (2013)"
+          }
+        ],
+        "rows": [
+          {
+            "name": "Score",
+            "values": [
+              null,
+              2.0,
+              29.55,
+              0.0,
+              0.0,
+              null,
+              1840.0,
+              0.69,
+              2.0
+            ]
+          },
+          {
+            "name": "State Ranking",
+            "values": [
+              13,
+              27,
+              4,
+              51,
+              70,
+              6,
+              30
+            ]
+          }
+        ]
+      },
+      {
+        "name": "Deforestation",
```

Comment:
It's static, but we considered the logic should be on the backend, because that's the pattern we followed for the Tool endpoints (old API). 
I think in the old API Clement has a workflow to insert data from a static JSON (like 'which indicators' belongs to which category') in the database.